.drone.yml

```diff
@@ -28,41 +28,105 @@ steps:
 
 - name: download testdata
   image: alpine/git:1.0.7
+  volumes:
+  - name: test-collections-v
+    path: /drone/src/testsuite/volumes/collections
   commands:
   - mkdir -p testsuite/volumes/collections
   - git clone https://github.com/hoover/testdata testsuite/volumes/collections/testdata
+  - chown 666:666 testsuite/volumes/collections/testdata
 
-- name: do the docker setup
-  image: docker/compose:1.24.0
+- name: build docker image
+  image: docker
   volumes:
   - name: dockersock
-    path: /var/run
+    path: /var/run/docker.sock
   commands:
-  - cd testsuite
-  - mkdir -p volumes/search-es/data
-  - chmod 777 volumes/search-es/data
-  - docker-compose up -d --scale snoop=0
+  - docker build . -f Dockerfile --tag liquidinvestigations/hoover-snoop2:test
 
-- name: py.test
-  image: docker/compose:1.24.0
+- name: run tests
+  image: liquidinvestigations/hoover-snoop2:test
+  pull: if-not-exists
   volumes:
   - name: dockersock
-    path: /var/run
+    path: /var/run/docker.sock
+  - name: gnupg-v
+    path: /opt/hoover/gnupg
+  - name: test-collections-v
+    path: /opt/hoover/collections
+  - name: snoop-blobs-v
+    path: /opt/hoover/snoop/blobs
+  # - name: search-v-testing
+  #   path: /opt/hoover/search/hoover/site/settings/testing_local.py
+  environment:
+    WAIT_HOSTS: search-es:9200, snoop-pg:5432
+    WAIT_HOSTS_TIMEOUT: 60
+    SNOOP_DB: "postgresql://snoop:snoop@snoop-pg:5432/snoop"
+    SNOOP_COLLECTIONS: '[{"name": "testdata", "process": true}]'
+    SNOOP_AMQP_URL: "amqp://snoop-rabbitmq"
+    SNOOP_TIKA_URL: "http://snoop-tika:9998"
+    SNOOP_ES_URL: "http://search-es:9200"
+    SNOOP_COLLECTION_ROOT: "/opt/hoover/collections"
+    SNOOP_BLOB_STORAGE: "/opt/hoover/snoop/blobs"
   commands:
-  - cd testsuite
-  - docker-compose run --rm snoop bash -c '/wait && py.test -vv'
+  - cd /opt/hoover/snoop
+  - py.test -vv
 
 services:
 - name: docker daemon in docker
   image: docker:dind
   privileged: true
   volumes:
   - name: dockersock
-    path: /var/run
+    path: /var/run/docker.sock
+
+- name: snoop-rabbitmq
+  image: rabbitmq:3.7.3
+
+- name: snoop-tika
+  image: logicalspark/docker-tikaserver
+
+- name: search-es
+  image: docker.elastic.co/elasticsearch/elasticsearch:6.2.4
+  volumes:
+  - name: search-es-v
+    path: /drone/src/volumes/search-es/data
+  environment:
+    discovery.type: single-node
+
+- name: snoop-pg
+  image: postgres:12.2
+  environment:
+    POSTGRES_USER: snoop
+    POSTGRES_DATABASE: snoop
+    POSTGRES_PASSWORD: snoop
+  volumes:
+  - name: snoop-pg-v
+    path: /drone/src/volumes/snoop-pg/data
+
 
 volumes:
 - name: dockersock
+  host:
+    path: /var/run/docker.sock
```

Comment:
You're making all tests use the same :test image on the host. What if we're running multiple tests on multiple branches at the same time?

Please leave this section as it was, maybe keep the `collections` volume